### PR TITLE
rename GaussianHeatMap to GaussianHeatmap

### PR DESCRIPTION
**Pull request by jin-s13**

### CODE REVIEW

File: mmpose/datasets/pipelines/top_down_transform.py [hunk 1]
```diff
@@ -253,14 +255,19 @@ def __init__(self,
...
-                 target_type='GaussianHeatMap',
+                 target_type='GaussianHeatmap',
                  encoding='MSRA',
                  unbiased_encoding=False):
         self.sigma = sigma
         self.unbiased_encoding = unbiased_encoding
         self.kernel = kernel
         self.valid_radius_factor = valid_radius_factor
-        self.target_type = target_type
+        if target_type == 'GaussianHeatMap':
```
Comment (innerlee) [inline mmpose/datasets/pipelines/top_down_transform.py:265]: Ask for opinion: is it safe to compare the lower cased string?

Comment (innerlee) [reply mmpose/datasets/pipelines/top_down_transform.py:265]: If so, we can avoid the deprecation warning

Comment (jin-s13) [reply mmpose/datasets/pipelines/top_down_transform.py:265]: The lower case `gaussianheatmap` looks strange.

Comment (innerlee) [reply mmpose/datasets/pipelines/top_down_transform.py:265]: Yeah, how about we do not use it but we allow it?

**Review by innerlee — APPROVED** [review]

File: mmpose/datasets/pipelines/top_down_transform.py (hunk 1 above)
Comment (jin-s13) [reply mmpose/datasets/pipelines/top_down_transform.py]: OK.

**Comment by ly015** [comment]
Please solve the conflict. And it would be great to have a unittest for BC-breaking.

**Comment by jin-s13** [comment]
Both `GaussianHeatMap` and `GaussianHeatmap` work.

**Pull request merged**